qucs/imagewriter.cpp

```diff
@@ -371,8 +371,8 @@ void ImageWriter::getSelAreaWidthAndHeight(Schematic *sch, int &wsel, int &hsel,
             pd->Bounding(x1,y1,x2,y2);
             updateMinMax(xmin,xmax,ymin,ymax,x1,x2,y1,y2);
 
-            for (auto& pg: qAsConst(pd->Graphs)) {
-                for (auto& pm: qAsConst(pg->Markers)) {
+            for (auto& pg: std::as_const(pd->Graphs)) {
```

Comment:
Need to check if `as_const` supported by GCC9 from Ubuntu-20.04. This distribution have to be supported.

Reply:
IMHO 
1. we just need to remove the function **qAsConst** and use C++ 11 (there are many places in qucs where Graphs iterated without **qAsConst**)
2.  for **setNamedColor** we can write single utility function with conditional compilation and use it everywhere instead off many 
#if QT_VERSION >= QT_VERSION_CHECK(6, 4, 0)

Reply:
Yes, the wrapper macro or function for `setNamedColor()` will be better solution. We can put it somewhere in `misc.h`